lib/protocon/plugins/driver_tcp.py

```diff
@@ -93,9 +97,16 @@ def open(self):
 
 		tcp_sock = socket.socket(self._addrinfo.family, self._addrinfo.type)
 		if self.settings['type'] == 'client':
+			if self.url.scheme.startswith('ssl'):
+				context = ssl.create_default_context()
+				context.check_hostname = False
+				context.verify_mode = ssl.CERT_NONE
+				tcp_sock = context.wrap_socket(tcp_sock)
```

Comment:
Let me know if you don't like modifying `tcp_sock` like this. I figured the code would be simpler this way.